aslite/db.py

```diff
@@ -130,12 +132,12 @@ def get_email_db(flag='r', autocommit=True):
     assert flag in ['r', 'c']
     edb = SqliteDict(DICT_DB_FILE, tablename='email', flag=flag, autocommit=autocommit)
     return edb
-def setup_chemical_embeddings_collection(client : MilvusClient):
-    
+
+def setup_chemical_embeddings_collection(client: MilvusClient):    
     schema = MilvusClient.create_schema(auto_id=False)
     schema.add_field(field_name="id", datatype=DataType.INT64, is_primary=True, auto_id=True)
     schema.add_field(field_name="chemical_embedding", datatype=DataType.BINARY_VECTOR, dim=config.chemical_embedding_size)
-    schema.add_field(field_name="paper_id", datatype=DataType.INT64)
+    schema.add_field(field_name="paper_id", datatype=DataType.VARCHAR, max_length=65535)
```

Comment:
jesteście pewni tego varchar(65535)? nie wiem jak działa milvus, to nie będzie Wam miejsca marnował?